MisusedInit.frost:3:9: error: cannot directly call 'init' outside of an init method
MisusedInit.frost:5:18: error: 'init' may only be called on 'self' or 'super'
MisusedInit.frost:11:18: error: 'init' may only be called on 'self' or 'super'
3 errors
